### PR TITLE
I added a notebook version

### PR DESCRIPTION
**Pull request by Moeinh77**
In this note book I sued exponential learning rate decay and also weight decay, which improved the result of model with 2 iterations and 64 8 16 16 layers to 92% accuracy

### CODE REVIEW

**Comment by Moeinh77** [comment]
The edited train.py now with the new settings can reach the to 92% accuracy on test set

**Comment by yl-1993** [comment]
Thanks for the effort. The modification of train.py looks good to me.
Could you please use more suggestive name for `.ipynb`? It is not easy to keep track of what `notebook model 222.ipynb` means.

**Comment by Moeinh77** [comment]
Hello
the note book is the model with setting 64 8 16 16 and K =3 and all the
three stages are trained for 2 iterations of EM. it is basically the
train.py code but in a notebook. However i added a couple of things to the
notebook that are not in the .py file like resuming the training from where
it was left off.

*------------------*
*Moein Hasani*

*find me :*

*LinkedIn account <https://www.linkedin.com/in/moein-hasani/>*

*Github account <https://github.com/Moeinh77>*

**Comment by Moeinh77** [comment]
I actually wanted to just commit the changes to .py file because the note
is the same as the .py with changes. So, you can ignore the notebook and
just accept the changes of the .py. and you can update the readme and
mention the 92% accuracy fo the model.

*------------------*
*Moein Hasani*

*find me :*

*LinkedIn account <https://www.linkedin.com/in/moein-hasani/>*

*Github account <https://github.com/Moeinh77>*

**Comment by yl-1993** [comment]
Hi, this PR will merge all the difference from `Moeinh77:master`, so I cannot just merge the .py file.

I suggest to 
(1) create a new branch in your repo, e.g., `dev`.
(2) revert `dev` branch to the commit `a2ed987`, add resuming in train.py and modify the README.
(3) create a PR from `dev`.
Then I think the PR will be ready for merging.
